python/ola/OlaClient.py

```diff
@@ -1293,7 +1293,11 @@ def _RDMMessage(self, universe, uid, sub_device, param_id, callback, data,
     request.uid.device_id = uid.device_id
     request.sub_device = sub_device
     request.param_id = param_id
-    request.data = data
+    if sys.version >= '3.2':
```

Comment:
Testing dotted versions with string comparison means that this will return False on the current version of Python, which was probably not the intention here.